### DISCUSSION TITLE
storybook github io 적용 관련 논의

### DISCUSSION THREAD

**Discussion by brightchul** [es6frontendstudy2021/cheese-ui-component#23] [Show and tell]
cheese-ui-component 관련 스토리북 github.io에 올렸습니다.  [[cheese-UI-component storybook]](https://es6frontendstudy2021.github.io/cheese-ui-component/)
현재는 관련 내용이 gh-pages에 있고, develop에 적용되지는 않은 상황입니다.

시험적으로 올렸기 때문에 한번 확인해 보시고 괜찮다 싶으면 아래 둘 중 하나로 하고 싶습니다.

1. develop에 해당 세팅을 적용하는 방식
2. gh-pages에 두고, develop이 업뎃 될때마다 merge 하면서 추적하는 방식

간단하게 의견 주시면 감사하겠습니다.

**Comment by cheese-cake-cookie** [comment]
안녕하세요 철훈님!
그럼 현재 적용해주신 방식은 설명해주신 것 중 2번에 해당하는 방식이라고 이해했는데 맞을까요?

1번 방식이 세팅을 해두면 적용 과정 자체는 자동화(라는 표현이 맞을까요?) 되어 번거로운 부분은 적을 것 같으나
초보자 관점에서 저는 스토리북 적용 방식이 좀 더 명확하게 보이는 2번 방식으로 해보는 게 좋을 것 같습니다!

**Reply by brightchul** [reply]
네 알겠습니다. 그러면 2번 방식으로 하겠습니다.
